### PR TITLE
EWL-5386 people listing page finalized

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- EWL-5386: SG2 | Create People Listing Page (Grid view only) Finalized Version](https://issues.ama-assn.org/browse/EWL-5386)

## Description
Add people listing page with people listing cards and masthead

## To Test
- [x] `gulp serve`
- [x] visit http://localhost:3000/?p=pages-people-listing
- [x] observe a masthead with two rows with 4 columns of cards
- [x] resize your browser to tablet size
- [x] observe three rows with 2 columns of cards 
- [x] resize your browser to mobile size
- [x] observe the cards are single stacked 
- [x] Did you test in IE 11?

## Visual Regressions
updated backstop test by adding people listing page 
added all people listing backstop image refs
The promo group image refs also had to be updated because a container class was added to prevent the from spilling outside of their containers

## Relevant Screenshots/GIFs
screen shot 2018-07-06 at 3 38 22 pm

screen shot 2018-07-03 at 10 51 45 am
screen shot 2018-07-03 at 10 51 51 am

## Remaining Tasks
N/A

## Additional Notes
This PR is branched off https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/314. #314 needs to get merged before this PR can be reviewed
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/05-pages/people-listing.json [hunk 1]
```diff
@@ -0,0 +1,1111 @@
...
+      },
+      "heading": {
+        "level": "1",
+        "text": "Organized Medical Staff Section (OMSS) Governing Council",
+        "class": "ama__h1"
+      },
+      "subtitle": {
+        "text": "Organized Medical Staff Section (OMSS) Governing Council"
+      },
+      "date": {
```
Comment (labbydev) [inline styleguide/source/_patterns/05-pages/people-listing.json]: There should not be a date on the masthead of this page.

**Review by labbydev — CHANGES_REQUESTED** [review]
None of the conflict of interest links include the download icon as dictated in the Zeplin. 

Some of the the items should show without full content so that we can test how they appear with different amounts of vertical content.

None of the items include the "View Full Profile" button some should and some should not so that we can test both ways. 

There do not seem to be backstop reference images for just the card pattern.

Please include the ticket and the testing instructions for any sub patterns as well.

**Comment by designerneil** [comment]
@labbydev I don't understand what you mean by "Please include the ticket and the testing instructions for any sub patterns as well."

**Comment by labbydev** [comment]
@designerneil what I mean, is that you are including code to create the sub patterns (the cards on the listing) in this PR that are not included in develop. However, you do not indicate that this PR is dependent on any other PR, nor do you include testing instructions or notes for those patterns.

**Comment by designerneil** [comment]
@labbydev the sub patterns testing instructions are included in PR#

EWL-5411 Add people listing page stub #313
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/313

EWL-5289 Add two column right full width #320 
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/320

EWL-5402 Create "People Listing" Molecule and CTA variant #314
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/314

**Review by labbydev — APPROVED** [review]

**Pull request merged**